Packet++/header/NtpLayer.h

```diff
@@ -0,0 +1,649 @@
+#ifndef PACKETPP_NTP_LAYER
+#define PACKETPP_NTP_LAYER
+
+#include "Logger.h"
+#include "IPv4Layer.h"
```

Comment:
Is this header used somewhere?

Reply:
In that case, I think we can move `#include "IPv4Layer.h"` to `NtpLayer.cpp` 🤔

Reply:
Replaced with `#include "IpAddress.h"` because of `void setReferenceIdentifier(IPv4Address val);` since we only need `IPv4Address`